### DISCUSSION TITLE
Pieces Sidebar on Edge reloading when switching a tab

### DISCUSSION THREAD

**Discussion by sunarkadeep** [pieces-app/support#149] [General]
Hello everyone! one thing that I am a bit disliking is the fact that with the pieces extension open in sidebar, whenever I switch tabs on Edge, the pieces in sidebar just reloads and goes back to the 'Discovered Snippets' window (which is like the starting window of pieces browser extension)...and so I just have to again go back to any snippet or copilot....any way to fix this or this sidebar thing is supposed to do like this? do all of you experience the same thing or is it different?

**Comment by rosie-at-pieces** [comment]
@rutvik-pieces thoughts?

**Comment by rutvik-at-pieces** [comment]
I'm able to reproduce this issue on edge. Will check into it. Thanx for reporting the issue @sunarkadeep 🙌
